src/NodeServices/WorkspaceEvents.cs

```diff
@@ -95,10 +104,16 @@ public WorkspacesModificationEventArgs(Guid id, string name, Type type)
     public class WorkspacesSettingsChangedEventArgs : EventArgs
     {
         public double ScaleFactor { get; private set; }
+        public bool EnableLegacyPolyCurveBehavior { get;private set; }
```

Comment:
does this need to be public for some reason?

Reply:
what is watching these events?

Reply:
Yes, this is used on the LibG side.